srcpkgs/nvtop/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'nvtop'
+pkgname=nvtop
+version=1.2.2
+revision=1
+archs="x86_64"
+build_style=cmake
+makedepends="ncurses-devel"
+depends="nvidia ncurses"
```

Comment:
You don't need to list `ncurses` in `depends`, it will be detected.

Reply:
Does this work with `nvidia390` as well? You could drop the `nvidia` requirement as well for those on older systems.  If you do that, set `archs="x86_64 i686"`.

Reply:
Hi, and thank you for review.
About `nvidia390`: nvtop uses NVML Library which comes with nvidia drivers and as developer stated in github readme:
"_The NVML library does not support some of the queries for GPUs coming before the Kepler microarchitecture. Anything starting at GeForce 600, GeForce 800M and successor should work fine_". 
So, i don't think that this package will be working correctly on `nvidia390` and their GPU models.